books/free-programming-books-langs.md

```diff
@@ -427,7 +427,7 @@ That section got so big, we decided to split it into its own file, the [BY SUBJE
 * [Beej's Guide to the GNU Debugger (GDB)](http://beej.us/guide/bggdb/) - B. Hall (HTML)
 * [Build Your Own Lisp](http://www.buildyourownlisp.com) - Daniel Holden
 * [C Elements of Style](http://www.oualline.com/books.free/style/) - Steve Oualline
-* [C for Python Programmers - Carl Burch](http://www.toves.org/books/cpy/)
+* [C for Python Programmers - Carl Burch](http://www.cburch.com/books/cpy/index.html)
```

Comment:
Maybe better this according to CONTRIBUTING url guidelines :wink:

```suggestion
* [C for Python Programmers - Carl Burch](http://www.cburch.com/books/cpy/)
```

Reply:
Sure. Will do the needful

Reply:
Hey. I have made the changes according to contributing url guidelines

Reply:
Not reflected yet in this PR branch.

To be sure that upstream is synced, submit changes over your forked branch: `Harshita-Kohli-patch-1`

Reply:
ok sure.

Reply:
Now that is submitted you can mark this conversation as resolved

Reply:
Ok great! Thanks

Reply:
Remember add `Resolves #6500` in PR description body

Reply:
Yes yes sure